### PR TITLE
Lizard Plushies Can Now "Weh!"

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Adds a "weh!" to the lizard plushie.
https://github.com/ParadiseSS13/Paradise/assets/143041327/648eba9a-9b63-4b1f-8971-1700fce71f14

## Why It's Good For The Game
Weh!

## Testing
Spawned a lizard plushie.
Hugged.
Weh!

## Changelog
:cl:
soundadd: Lizard plush can now go "weh!"
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -809,6 +809,16 @@
...
 	icon_state = "plushie_lizard"
 	item_state = "plushie_lizard"
+	var/cooldown = 0
+
+/obj/item/toy/plushie/lizardplushie/attack_self(mob/user)
+	if(!cooldown)
+		playsound(user, 'sound/voice/weh.ogg', 10, 0)
+		visible_message("<span class='danger'>Weh!</span>")
+		cooldown = 1
+		spawn(30) cooldown = 0
```
Comment (GDNgit) [inline code/game/objects/items/toys.dm:819]:
```suggestion
		spawn(30) cooldown = 0
```
come on

**Review by GDNgit — CHANGES_REQUESTED** [review]
april fools was 11 days ago mate

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (Fordoxia) [reply code/game/objects/items/toys.dm:819]: Stolen from the vox plush code.

Comment (DGamerL) [reply code/game/objects/items/toys.dm:819]: That doesn't mean it's good like that, at the very least use a varset callback for this

Comment (GDNgit) [reply code/game/objects/items/toys.dm:819]: Replace this with a varset callback then and make this true/false bools

Comment (Contrabang) [reply code/game/objects/items/toys.dm:819]: Even better, dont use a timer and use the tg_cooldowns instead

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (Fordoxia) [reply code/game/objects/items/toys.dm:819]: Time to refactor all the plushies.

Comment (Fordoxia) [reply code/game/objects/items/toys.dm]: > Even better, dont use a timer and use the tg_cooldowns instead

It is done.

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -809,6 +809,15 @@
 	desc = "An adorable stuffed toy that resembles a lizardperson."
 	icon_state = "plushie_lizard"
 	item_state = "plushie_lizard"
+	COOLDOWN_DECLARE(hug_sound)
+
+/obj/item/toy/plushie/lizardplushie/attack_self(mob/user)
+	if(!COOLDOWN_FINISHED(src, hug_sound))
+		..()
+		return
+	playsound(user, 'sound/voice/weh.ogg', 10, 0)
```
Comment (Burzah) [inline code/game/objects/items/toys.dm:818]:
```suggestion
	playsound(user, 'sound/voice/weh.ogg', 10, FALSE)
```

**Review by Burzah — CHANGES_REQUESTED** [review]

**Comment by SynthTwo** [comment]
Why 'weh'? Hiss makes more sense.

**Comment by ArcticValor** [comment]
> Why 'weh'? Hiss makes more sense.

cause funny SS14 meme

**Comment by AffectedArc07** [comment]
If I still had veto rights this PR would get it in a heartbeat. 

Maybe I should unretire.

**Review by DGamerL — CHANGES_REQUESTED** [review]
>Sound PR
>No sound on the PR desc

Please fix

**Comment by Fordoxia** [comment]
> > Sound PR
> > No sound on the PR desc
> 
> Please fix

Fixed.

**Comment by FunnyMan3595** [comment]
Silly things are good, but this really doesn't fit a lizard IC-ly, and the sound effect doesn't really fit the rest of our sounds, which are more serious. I'm against.

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -809,6 +809,15 @@
 	desc = "An adorable stuffed toy that resembles a lizardperson."
 	icon_state = "plushie_lizard"
 	item_state = "plushie_lizard"
+	COOLDOWN_DECLARE(hug_sound)
+
+/obj/item/toy/plushie/lizardplushie/attack_self(mob/user)
+	if(!COOLDOWN_FINISHED(src, hug_sound))
+		..()
+		return
```
Comment (Contrabang) [inline code/game/objects/items/toys.dm:817]:
```suggestion
		return ..()
```

**Comment by ATP-Engineer** [comment]
image

**Comment by TORE01** [comment]
Nice PR. Not sure weh people dislike it.

**Comment by S34NW** [comment]
You currently have 3 design members objecting, and no-one in favour.

Here is some feedback from the team:

- Silly things are good, but this really doesn't fit a lizard IC-ly, and the sound effect doesn't really fit the rest of our sounds, which are more serious.
- The sound is just plain off-putting. Also I swear the idea of this and this exact sound came up before in the past and was denied.
- I agree with the above

**Pull request closed without merging**